insights/tests/client/apps/test_compliance.py

```diff
@@ -156,11 +192,32 @@ def test_tailored_file_is_not_downloaded_if_tailored_is_missing(config):
 
 @patch("insights.client.apps.compliance.open", new_callable=mock_open)
 @patch("insights.client.config.InsightsConfig")
-def test_tailored_file_is_downloaded_if_needed(config, call):
+def test_tailored_file_is_downloaded_from_initial_profile_if_os_minor_version_is_missing(config, call):
```

Comment:
Notice here the test doesn't need to change at all if the `os_minor_version` attribute is missing from the API response. I also added a check for `'tailored': False` for good measure.